tests/unit/resources/test_policies.py

```diff
@@ -141,6 +141,20 @@ def test_get_vms_not_found(self, mock_get):
 
         mock_get.assert_called_once_with('/policies/ABCDE/virtual_machines')
 
+    @mock.patch.object(Connection, "post")
+    @mock.patch.object(Connection, "get")
+    def test_create_policy(self, mock_post, mock_get):
+        resource_data = [{'name': 'test'}]
+        mock_post.return_value = {policies.DATA_FIELD: resource_data}
```

Comment:
the connection post should be returning a list of affect objects

mock_post.return_value = None, [{'object_id': '12345'}]

the mock_get should be returning the policies

Reply:
Updated the test as per the review comment